projects/lsp4mp/projects/maven/microprofile-graphql/pom.xml

```diff
@@ -39,6 +39,11 @@
         <artifactId>smallrye-graphql-client-api</artifactId>
         <version>1.0.2</version>
     </dependency>
+    <dependency>
```

Comment:
Perhaps it should be better to have 2 projects microprofile-graphqwith Quarkus 3.1 and Quarkus <3.1 since we have different validation kind?